src/main/webapp/app/admin/metrics/blocks/metrics-system/metrics-system.component.ts

```diff
@@ -7,7 +7,6 @@ import { DatePipe, DecimalPipe } from '@angular/common';
     selector: 'jhi-metrics-system',
     templateUrl: './metrics-system.component.html',
     changeDetection: ChangeDetectionStrategy.OnPush,
-    standalone: true,
```

Comment:
i might be missing something but why is this removed? also, it seems that no new `standalone: true` have been added